cli/tests/integrations/test_task.py

```diff
@@ -184,7 +184,7 @@ def test_log_two_tasks():
     assert stderr == b''
 
     lines = stdout.decode('utf-8').split('\n')
-    assert len(lines) == 23
+    assert len(lines) == 19
```

Comment:
i agree with you, this is fragile indeed, however, on the other hand, this assures that we expect exactly 19 lines nothing more nothing less. If you look through the CLI code base, we use similar test all over the places.